api/server.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express");
+
+const server = express();
+
+server.use(express.json());
+
+const games = [];
+
+let id = 0;
+
+//POST Endpoint
+server.post("/games", (req, res) => {
+  const { title, genre, releaseYear } = req.body;
+  id++;
```

Comment:
It would be best to increment the id only if you are sure it's being added to the database.